docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml

```diff
@@ -42,7 +42,7 @@ return currentListSize
 
         return hasQueryTextInClient
         opt hasQueryTextInClients
-            CommandFindClient->Ui:displayOneClient(Client: currentClient, int: clientIndex)
+            CommandFindClient->Ui:displayOneClient(currentClient: Client, clientIndex: int)
             activate Ui #CDEFF9
             return
```

Comment:
According to your code, there is a check
```
if (hasNoMatches) {
    ui.showToUser(MESSAGE_NO_CLIENT_MATCHES);
}
```
before the method returns. Perhaps it would be more accurate if you included that in your diagram, or put a note for the reader that says you have omitted that.